var/spack/repos/builtin/packages/hard/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Hard(CMakePackage):
+    """A FleCSI-based radiation-hydrodynamics solver suite
+    for the study of astrphysical phenomena"""
+
+    git = "https://github.com/lanl/hard"
+
+    version("develop", branch="main")
+
+    variant("catalyst", default=False, description="Enable catalyst for paraview interface")
+    variant("radiation", default=True, description="Enable support for radiation physics")
+    variant("tests", default=False, description="Enable unit tests")
+
+    depends_on("flecsi@2.3.0")
```

Comment:
```suggestion
    depends_on("flecsi@2.3:")
```

Reply:
I had it like that initially but then it picks `2.3-dev` which has some issues to compile. The code works is set for `2.3.0` for now.